lib/TabTracker.js

```diff
@@ -45,6 +52,16 @@ TabTracker.prototype = {
     }
     tabs.removeListener("open", this.onOpen);
     Services.obs.removeObserver(this.onPerfLoadComplete, PERF_LOG_COMPLETE_NOTIF);
+    Services.obs.removeObserver(this.onCacheUpdate, CACHE_UPDATE_NOTIF);
```

Comment:
Not having the cache turned on will result in the data not being populated. It makes for an non-obvious coupling with an unrelated pref state, even if in all likelihood, that pref will not be tampered with.

We want to minimize such couplings.